app/experimenter/experiments/constants.py

```diff
@@ -897,7 +902,7 @@ class ExperimentConstants(object):
 
     BUGZILLA_ADDON_TEMPLATE = (
         """
-    Experiment Type: Opt-Out experiment
+    Experiment Type: AddOn experiment
```

Comment:
I think it's "Add-on":

<img width="325" alt="Screenshot 2019-05-21 17 20 23" src="https://user-images.githubusercontent.com/119884/58131463-bcb53000-7bec-11e9-8741-a74b9f09086d.png">